.storybook/config.js

```diff
@@ -4,6 +4,7 @@ import { withInfo } from "@storybook/addon-info";
 
 // Import css
 import "../src/css/teamsnap-ui.scss";
+// import "../src/css/themes/custom-aquablue.scss";
```

Comment:
This will be used for development to ensure that theming can still work with our changes.